presto-docs/src/main/sphinx/release/release-0.151.rst

```diff
@@ -0,0 +1,34 @@
+=============
+Release 0.151
+=============
+
+General Changes
+---------------
+
+* Fix bug where partial aggregation may be repeated when ``task.concurrency = 1``.
+* Fix query failure when ``array``, ``map``, or ``row`` type is used in non-equi ``JOIN``.
+* Fix performance regression for queries using ``OUTER JOIN``.
+* Fix query failure when using the :func:`arbitrary` aggregation function on ``integer`` type.
+* Add various math functions that operate directly on ``float`` type.
+* Add flag ``experimental.legacy-array-agg`` to restore legacy :func:`array_agg`
+  behavior (ignore ``NULL`` input).
+* Add support for uncorrelated ``EXISTS`` clause.
+* Add :func:`cosine_similarity` function.
+* Allow Tableau connector to use catalogs other than ``hive``.
+
+Verifier Changes
+----------------
+
+* Add ``shadow-writes.enabled`` option which can be used to rewrite ``CREATE TABLE AS SELECT``
+  queries, so that they write to a temporarily table.
```

Comment:
How about
```
* Add ``shadow-writes.enabled`` option which can be used to transform ``CREATE TABLE AS SELECT``
  queries to write to a temporary table (rather than the originally specified table).